setup.py

```diff
@@ -5,56 +5,60 @@
 setuptools.setup(
     name=__name__,
     version=__version__,
-    description='Object browser for OS_swift API',
+    description="Object browser for OS_swift API",
     author=__author__,
     author_email="sampsa.penna@csc.fi",
```

Comment:
would be good to have our support email here or leave out.

Reply:
True, removing the support email for now – it can be added back later

Reply:
As I think that the support for software itself should be gotten on GitHub, and other (mainly usage) support should be given in the actual service